tests/models/test_selector.py

```diff
@@ -168,16 +168,23 @@ def test_models_selector_model_selector_model_rules(model_rules, rules):
     [
         (ServerEventFactory(), ServerEventModel),
         (PageCloseBrowserEventFactory(), PageCloseBrowserEventModel),
-        ({"invalid": "event"}, None),
     ],
 )
-def test_models_selector_model_selector_get_model(event, model):
-    """Given an event the ModelSelector.get_model method should return the corresponding model."""
+def test_models_selector_model_selector_get_model_with_valid_event(event, model):
+    """Given a valid event the get_model method should return the corresponding model."""
```

Comment:
```suggestion
    """Tests given a valid event the get_model method should return the corresponding model."""
```

Reply:
When comparing with test_validator docstrings, I think it is better to get the same sentence structure. WDYT?

Reply:
Good catch) Thanks)